src/Store/VimStoreConnector.re

```diff
@@ -1122,20 +1122,19 @@ let start =
         copyActiveFilepathToClipboardEffect,
       )
 
-    | VimDirectoryChanged(directory) =>
+    | VimDirectoryChanged(workingDirectory) =>
```

Comment:
When we change workspaces - do we load persisted settings for the new workspace?

Reply:
It doesn't, but it could. It would be kind of weird to have the window position and size suddenly change just because you `:cd` somewhere. But if we want other settings to apply at that point, we certainly could. The problem is more how to detect that the workspace changed perhaps. Could do it on `DirectoryChanged` I guess.

Reply:
Good point. I guess there are two classes of settings - ones that you would only want applied initially, and ones that you'd want to be applied anytime... Since we only have the one 'class' today, we can defer this until we have persisted settings that have a need for this.